### PR TITLE
Aztec: Update preferences toggle

### PR DESCRIPTION
**Pull request by frosty**
This PR updates the App Settings preferences toggle for Aztec.

simulator screen shot 27 jun 2017 16 17 41

Note: The button in the footer currently doesn't link anywhere.

To test:

* Launch the app, visit App Settings
* Check the correct current editor setting is reflected in the options
* Try changing between the different options and check the correct editor is presented
* Ensure the 'help' footer appears if you select the Beta option, and disappears if you toggle a different option.
* Check the changes to `ImmuTable` footer delegate methods didn't affect other VCs that use `ImmuTable`.

Needs review: @SergioEstevao 
cc @iamthomasbishop

### CODE REVIEW

**Comment by frosty** [comment]
Just pushed a quick change to the analytics. I was previously triggering 'toggled on' when `hybrid` was enabled, but we only want it for `aztec`.

File: WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift [hunk 1]
```diff
@@ -265,3 +324,42 @@ fileprivate struct MediaSizingRow: ImmuTableRow {
...
+        stackView.distribution = .equalSpacing
+        stackView.axis = .horizontal
+
+        let label = UILabel()
+        label.font = UIFont.preferredFont(forTextStyle: .footnote)
+        label.text = NSLocalizedString("Editor beta release notes & bug reporting", comment: "Label for button linking to release notes and bug reporting help for the new beta Aztec editor")
+        label.textColor = WPStyleGuide.greyDarken10()
+        stackView.addArrangedSubview(label)
+
+        let button = UIButton(type: .custom)
```
Comment (SergioEstevao) [inline WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift]: any reason for this to be a button? or can it be just an UIImageView?

At the moment you can tap it, but it doesn't do anything...

Comment (SergioEstevao) [inline WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift]: I read your note on the PR, and I see the button will have an action in the future. Will change to approved.

**Comment by frosty** [comment]
Thanks @sergioestevao!

**Pull request merged**